core/vm/evm.go

```diff
@@ -66,7 +66,8 @@ func run(evm *EVM, contract *Contract, input []byte, readOnly bool) ([]byte, err
 					} else {
 						log.Debug("Calling Known Contract", "ID", evm.Id, "Name", name, "Method", method.RawName)
 						if method.RawName == "ovmREVERT" {
-							log.Debug("Contract Threw Exception", "ID", evm.Id, "asciified", string(input))
+							hex := hexutil.Encode(input)
```

Comment:
FWIW I intentionally put this as `asciified` and not hex because it is usually a solidity revert string which is readable ascii.  @smartcontracts and I ran into a case the other day where we wanted raw hex though, could be worth having both.

Reply:
I noticed it printing garbage characters so there must be a case where it does not include ascii

Reply:
It prepends the methodID hash for `Error(string)`, so there are consistently those garbage characters at the front.  There are definitely cases where there will be raw hex here but almost all normal contract reversions will be ascii.  I'd recommend that we have both.